### ISSUE TITLE
Syntax highlighting

### ISSUE THREAD

**Issue by Destructible-Watermelon** [covfefe-lang/covfefe#7]
We should definitely make a thing that highlights covfefifiable words to make it easier. other things too, like highlighting variables

**Comment by anna328p** [comment]
Yes. I have no idea how. Maybe someone could make a Vim syntax highlighting thing?
